src/components/call/CallGeneralInfo.tsx

```diff
@@ -54,17 +54,29 @@ const CallGeneralInfo: React.FC = () => {
           data-cy="end-date"
         />
       </MuiPickersUtilsProvider>
-      {templates && templates.length > 0 && (
-        <FormikDropdown
-          name="templateId"
-          label="Call template"
-          items={templates.map(template => ({
-            text: template.name,
-            value: template.templateId,
-          }))}
-          InputProps={{ 'data-cy': 'call-template' }}
-        />
-      )}
+      <FormikDropdown
+        name="templateId"
+        label="Call template"
+        loading={loadingTemplates}
+        noOptionsText="No templates"
+        items={templates.map(template => ({
+          text: template.name,
+          value: template.templateId,
+        }))}
+        InputProps={{ 'data-cy': 'call-template' }}
+      />
+      <FormikDropdown
+        name="proposalWorkflowId"
+        label="Proposal workflow"
+        loading={loadingProposalWorkflows}
+        noOptionsText="No proposal workflows"
+        items={
+          proposalWorkflows.length > 0
+            ? proposalWorkflowsWithInjectedSelectionRemoval
+            : []
+        }
+        InputProps={{ 'data-cy': 'call-workflow' }}
```

Comment:
@martin-trajanovski Change#1 I added data-cy to InputProps so they end up being on the dropdown

Reply:
This is good I wasn't aware of adding props on FormikDropdown like this. 🟢